terraform/provider/provider.go

```diff
@@ -200,13 +222,17 @@ func getConfigFromProfile(d *schema.ResourceData) (*client.Config, error) {
 		}
 	}
 
+	logrus.Debugf("loading profile (name: %s, dir: %s, default if empty)", name, dir)
```

Comment:
Let's structure logs as proper english sentences,please also use `logrus.WithError` and `logrus.WithFields`